public/app/features/plugins/plugin_loader.ts

```diff
@@ -115,7 +115,7 @@ exposeToPlugin('app/core/utils/kbn', kbn);
 exposeToPlugin('app/core/utils/ticks', ticks);
 exposeToPlugin('app/core/config', config);
 exposeToPlugin('app/core/time_series', TimeSeries);
-exposeToPlugin('app/core/time_series2', TimeSeries);
```

Comment:
😬 this is all legacy anyway.  If we change how it is exposed, we should just remove it

Reply:
We should not change this part (path imported by plugins)

Reply:
Thankyou for the feedback! So, not change here?